web/cypress/integration/route/search-route.spec.js

```diff
@@ -17,18 +17,20 @@
 /* eslint-disable no-undef */
 
 context('Create and Search Route', () => {
-  const timeout = 500;
+  const timeout = 1000;
```

Comment:
BTW @iamayushdas , why increase the `timeout` here?

Reply:
because in 500ms, all the components were not loading, so i increased i gave it sweet 1 second

Reply:
is there any thing else ?

Reply:
> because in 500ms, all the components were not loading, so i increased i gave it sweet 1 second, and it was giving a error

may be we can  analyze the cause of the error in your env , it seems that `timeout=500` works well in my local several times, and the code associated with timeout seems not modified in this pr

Reply:
Just for notes: the difference in timeout may be due to different computer performance. Also, cypress does not officially recommend using `cy.wait()`,  which can make the test unstable.

Reply:
cool @LiteSun